src/main/java/techItEasy/techItEasy/Repositories/TelevisionsRepository.java

```diff
@@ -0,0 +1,11 @@
+package techItEasy.techItEasy.Repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import techItEasy.techItEasy.Models.Televisions;
+
+import java.util.Optional;
+
+
+public interface TelevisionsRepository extends JpaRepository <Televisions, Integer>{
+    Optional<Televisions> findByBrand(String brand);
```

Comment:
Maak hier een List ipv een Optional van (denk ik). Als ik het goed begrepen heb, gebruik je een optional als je bijv een televisie uit de database opvraagt op basis van een id. Omdat je nog niet weet of dat id en die tv dus bestaat wil intellij dat je er eerst een optional van maakt.